src/Config/Schema/Validator/CompressionLevel.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Config\Schema\Validator;
+
+use Magento\MagentoCloud\Config\Validator\ResultFactory;
+use Magento\MagentoCloud\Config\Validator\ResultInterface;
+
+/**
+ * Validates the compression level value
+ */
+class CompressionLevel implements ValidatorInterface
```

Comment:
Maybe better to create one `RangeValidator` for all validators which need to check if the value in the given range and configure range in .yaml file
Now we have at least two variables with such validations.

Reply:
It's kindy tricky to pass the range to such class, we can have this as possible improvement